app/main/views/services.py

```diff
@@ -187,57 +187,56 @@ def start_new_draft_service(framework_slug, lot_slug):
     )
 
     section = content.get_section(content.get_next_editable_section_id())
+    if section is None:
+        section = content.get_section(content.get_next_section_id(None))
```

Comment:
I have tested it locally. It is fine.
<img width="1175" alt="screen shot 2017-02-13 at 10 38 56" src="https://cloud.githubusercontent.com/assets/2920760/22880079/b2b1df5e-f1d8-11e6-9ec2-78af7f8c6f8e.png">